clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/cache/client/AbstractCacheClient.java

```diff
@@ -65,6 +62,11 @@ public Collection<T> getAll(String account, String region) {
     return convertAll(data);
   }
 
+  public Collection<T> getAll(Collection<String> identifiers) {
+    Collection<CacheData> allData = cacheView.getAll(keyNamespace, identifiers);
+    return convertAll(allData);
+  }
+
```

Comment:
NOT entirely sure why we need this method... there's a bit of... is this the right place for this or inline into the caller.  IF doing this at least method name should match "getAllForIdentifiers" or similar.